server/cli/start.go

```diff
@@ -145,7 +145,9 @@ func runStart(cmd *commander.Command, args []string) {
 	}
 
 	log.Info("Starting cockroach cluster")
-	s, err := server.NewServer(Context)
+	stopper := util.NewStopper()
+	stopper.AddWorker()
```

Comment:
This AddWorker doesn't seem to be paired with a `go` statement or a `SetStopped`.